docs/tempo/website/getting-started/_index.md

```diff
@@ -40,29 +53,32 @@ information from a client application with minimal manual instrumentation of the
 * [OpenTemeletry .NET Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
 * [OpenTemeletry Python Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib)
 
-> Note: Check out our [instrumentation examples]() to learn how to instrument your
+> Note: Check out our [instrumentation examples]({{< relref "../guides/instrumentation" >}}) to learn how to instrument your
 > favourite language for distributed tracing.
 
-## 2. Grafana Agent
+## 2. Grafana Agent (Tracing pipeline)
 
 Once your application is instrumented for tracing, the next step is to send these traces
-to a backend for storage and visualization. The Grafana Agent is a service that is
-deployed close to the application, either on the same node or within the same cluster
-(in kubernetes) to quickly offload traces from the application and forward them to a storage
-backend. It also abstracts features like trace batching and backend routing
-away from the client. 
+to a backend for storage and visualization. It is common to build a tracing pipeline that 
+offloads spans from your application, buffers them and eventually forwards them to a backend. Tracing
+pipelines are optional (most clients can send directly to Tempo), but you will find that
+they become more critical the larger and more robust your tracing system is.
+
+The Grafana Agent is a service that is deployed close to the application, either on the same node or 
+within the same cluster (in kubernetes) to quickly offload traces from the application and forward them to 
+a storage backend. It also abstracts features like trace batching and backend routing away from the client. 
```

Comment:
I like how the OpenTelemetry docs mention [Agent and Gateway deployments](https://opentelemetry.io/docs/collector/getting-started/#deployment).

Maybe it's interesting to mention both ways here and throw some lines about the benefits of each one and how they can work together.

Or maybe not. Just throwing random thoughts :D

Reply:
It's a good idea, but if we start getting into agent/collector deployment strategies i think we'd that on its own page. It could be a very lengthy topic.